docker-compose.yml

```diff
@@ -53,6 +57,8 @@ services:
       - --db-schema=public # Once tooling-gql is deprecated, we can change the schema name if required
       # - --force-clean # To remove all tables and schema before starting the app
       - --timestamp-field # Enables created_at and updated_at in schema
+      - --disable-historical=true
+      # - --workers=4 # Number of available CPU cores strictly limits the usage of worker threads. Read more here - https://academy.subquery.network/run_publish/references.html#w-workers
```

Comment:
I'm still not confident with workers so have kept it commented (Can remove as well). 

I was testing some blocks in my local and made it fail at a particular block. SQ kept retrying 3-4 times and altogether skipped the block.

Reply:
What do you mean skipped the block? Like it didn't process it and moved onto the next?

That seems bad since it if had createAsset call then it would crash whenever something happened with the Asset

Reply:
Moved on to the next one